bin/httpd_configmap_generator

```diff
@@ -84,4 +93,4 @@ supported auth_type: #{HttpdConfigmapGenerator.supported_auth_types.join(', ')}
   end
 end
 
-HttpdConfigmapGenerator::Cli.new.run
+HttpdConfigmapGenerator::Cli.new.run if $PROGRAM_NAME == __FILE__
```

Comment:
why was this needed ?

Reply:
I needed to add that so I could load the file into the spec for testing without running the `#run` method at load time.